app/jobs/process_text_message_attachments_job.rb

```diff
@@ -0,0 +1,26 @@
+class ProcessTextMessageAttachmentsJob < ApplicationJob
+  retry_on NoMethodError
```

Comment:
this is nice! if we wanted to be even more narrow (so that we don't retry on other things weirdly being nil) we could define an empty exception class (`MissingAttachmentError`?) that inherits from StandardError and raise that when attachment[:body] is empty.